src/_posts/databases/mongodb/2000-01-01-compass.md

```diff
@@ -1,72 +1,146 @@
-internet. More information in the [dedicated page]({% post_url
-platform/databases/2000-01-01-access %}).
+Databases hosted on Scalingo are not by default directly available on Internet.
+There are a couple of ways to get access to a Scalingo MongoDB hosted database
+with MongoDB Compass:
+* Connection via an Encrypted Tunnel
+* Connection via the DB Tunnel of our CLI
+* Connection with TLS
 
 ## Configuration of Compass
 
-### Connection via an Encrypted Tunnel
-
-The connection data can be found in the `MONGO_URL` environment variable of your application.
+The connection URI can be found in the `MONGO_URL` environment variable of your
+application from the [web dashboard](https://dashboard.scalingo.com) or from the CLI:
 
-```
+```bash
 $ scalingo --app my-app env | grep MONGO_URL
-MONGO_URL=mongodb://sample-node-meanjs-7093:EsEjseivpacatVoogfijbiapgadTyg@c393a9e3-42fe-4e33-9e6c-8ee815e9af88.sample-node-meanjs-7093.mongodb.a.osc-fr1.scalingo-dbs.com:31312/sample-node-meanjs-7093
+MONGO_URL=$SCALINGO_MONGO_URL
+SCALINGO_MONGO_URL=mongodb://my-app-7093:EsEjseivpacatVoogfijbiapgadTyg@c393a9e3-42fe-4e33-9e6c-8ee815e9af88.my-app-7093.mongo.a.osc-fr1.scalingo-dbs.com:31312/my-app-7093?replicaSet=my-app-7093-rs0&ssl=true
 ```
 
 In this case:
-
-* Hostname: c393a9e3-42fe-4e33-9e6c-8ee815e9af88.sample-node-meanjs-7093.mongodb.a.osc-fr1.scalingo-dbs.com
+* Hostname: c393a9e3-42fe-4e33-9e6c-8ee815e9af88.my-app-7093.mongo.a.osc-fr1.scalingo-dbs.com
 * Port: 31312
-* User: sample-node-meanjs-7093
+* User: my-app-7093
 * Password: EsEjseivpacatVoogfijbiapgadTyg
-* Database: sample-node-meanjs-7093
-
-Fill the fields accordingly:
+* Database: my-app-7093
 
-{% assign width = "450px" %}
-{% assign img_url = "https://cdn.scalingo.com/documentation/compass/new_connection.png" %}
-{% include mdl_img.html %}
-
-The "SSH Tunnel" section must "Use Identity File". The identity file to provide
-is your private SSH key you uploaded on your Scalingo profile.
+### Connection via an Encrypted Tunnel
 
-The "SSH Hostname" and "SSH Port" depends on the region your database is
-deployed on:
+Open MongoDB Compass, then click on "New Connection".
+After that click on "Advanced Connection Options" and fill the fields
+accordingly:
+
+* General
+    * Connection String Scheme: mongodb
+    * Host: c393a9e3-42fe-4e33-9e6c-8ee815e9af88.my-app-7093.mongo.a.osc-fr1.
+    scalingo-dbs.com:31312 (Hostname + Port)
+* Authentication
+    * Authentication Method: Username / Password
+    * Authentication Mechanism: Default
+* TLS/SSL
+    * SSL/TLS Connection: On
+    * Certificate Authority (.pem): Depends on the region where your database
+    is hosted. You can find urls of our CA certificates at the end of the page.
+    and use it here
+* Proxy/SSH Tunnel
+    * SSH Tunnel/Proxy Method: SSH with Identity File
+    * SSH Hostname/SSH Port/SSH Username: Depends on the region where your
+    database is hosted. You can find all information at the end of the page.
```

Comment:
> at the end of the page

Which page are you talking about?

Reply:
Can you add the name of the section where this information is readable?

Reply:
Is it better with a link ?